textproc/moar/Portfile

```
@@ -1,61 +1,12 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
-PortGroup           golang 1.0
 
-go.setup            github.com/walles/moar 1.31.8 v
-go.offline_build    no
-revision            0
-
-conflicts           MoarVM
-
-description         Moar is a pager. It's designed to just do the right thing \
-                    without any configuration.
-
-long_description    Moar should work as a drop-in replacement for Less. \
-                    It features syntax highlighting, ANSI color coded text \
-                    support, incremental search and automatic decompression, \
-                    among others.
-
-checksums           rmd160  12e541c5ad34380a29e4366049b68949e2729af9 \
-                    sha256  77943278d1ca22c3ca405c6a227e4807ce8637b807c40802c739f901ae980151 \
-                    size    2849478
+replaced_by         moor
+PortGroup           obsolete 1.0
 
+name                moar
+version             1.31.8
+revision            0
```

`version` or `revision` must be greater than before to cause replacement to take place.
```suggestion
revision            1
```